src/main/java/com/sureit/admin/attendance/service/EmployeeService.java

```diff
@@ -51,13 +49,23 @@ public boolean createEmployee(Employee emp) {
 				employeeDAO.save(emp);
 				status = true;
 			}
-		} catch (DataAccessException e) {
+		} catch (Exception e) {
 			// TODO: handle exception
 
 		}
 		return status;
 	}
 
+	public Employee fetchEmployeeByEmailAddress(String emailAddress) {
+		Employee employee = null;
+		try {
+			employee = employeeDAO.findByEmailAddress(emailAddress);
+		} catch (Exception e) {
+			e.printStackTrace();

```

Comment:
Dont do console print in case of exception. return correct message to end user.
Handle Exception correctly.